sites-config/ci_config.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
for my own understanding, how does SGS know to look into sites-config/ folder? I see other SGS sites have a top level `config.json`, and the fields in there would point to different scripts in the ci/ folder, to configure how the build, install, serve steps should look like

Reply:
I'm not really sure about the details, other than the fact that the new system (Yext Sites) checks for the config file in the `sites-config` folder. The older system checks for the top level `config.json` (it can have other names as well). If we wanted to, we could create a ci folder just like the old setup

Reply:
Oh I see. I recently created a yext site, but following the old system approach. Was wondering if the system just check for files both location somehow, or prioritize the files from new approach. But either setup is fine with me!

Reply:
In Storm, when creating a site, it asks if you want an old version or a new one. I believe that's how it determines which config files to use